<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsqmlEditor</class>
 <widget class="QWidget" name="CwmsqmlEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Benutzerdefinierte Oberflächen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="JSEdit" name="m_pqteCode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_pqpbPropeties">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Eigenschaften</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="m_pqleName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Klasse</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="m_pqleClass"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="m_pqpbSelectClass">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../WMSadmin/resources/emresources.qrc">
            <normaloff>:/icons/search24.png</normaloff>:/icons/search24.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="m_pqchbDefault">
          <property name="text">
           <string>Standard</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Dialogtyp</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="m_pqrbObject">
             <property name="text">
              <string>Objekt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="m_pqrbObjectContainer">
             <property name="text">
              <string>Objektcontainer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="m_pqrbWmsContext">
             <property name="text">
              <string>WMS Kontext</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Einsatz</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="m_pqchbDesktop">
             <property name="text">
              <string>Desktop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="m_pqchbMobile">
             <property name="text">
              <string>Mobile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="m_pqchbTablet">
             <property name="text">
              <string>Tablet</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="m_pqchbWeb">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Web</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Formularart</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QRadioButton" name="m_pqrbDialog">
             <property name="text">
              <string>Dialog</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="m_pqrbWidget">
             <property name="text">
              <string>Widget</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="m_pqrbMainWindow">
             <property name="text">
              <string>Hauptfenster</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="m_pqpbOpenInExternalEditor">
            <property name="toolTip">
             <string>In externem Editor öffnen</string>
            </property>
            <property name="statusTip">
             <string>In externem Editor öffnen</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../WMSadmin/resources/emresources.qrc">
              <normaloff>:/icons/noticetypes.png</normaloff>:/icons/noticetypes.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JSEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">jsedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../WMSadmin/resources/emresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqpbOpenInExternalEditor</sender>
   <signal>clicked()</signal>
   <receiver>CwmsqmlEditor</receiver>
   <slot>OpenInExternalEditorSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>823</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SelectClassClickedSlot()</slot>
  <slot>PreviewClickedSlot()</slot>
  <slot>OpenInExternalEditorSlot()</slot>
  <slot>SaveClickedSlot()</slot>
 </slots>
</ui>
